smartcontractkit/external-adapters-js
# Fix/source map support

## Description
@karen-stepanyan and I identified an error when trying to run tests with `yarn test`:
```
  ● Test suite failed to run

    Cannot find module 'source-map-support' from '/Users/austinborn/chainlink/external-adapters-js/.yarn/cache/@cspotcode-source-map-support-npm-0.7.0-456c3ea2ce-9faddda775.zip/node_modules/@cspotcode/source-map-support'

      at resolveSync (.yarn/cache/resolve-patch-b4a69197d3-a0dd7d16a8.zip/node_modules/resolve/lib/sync.js:102:15)
```
This appears to be a regression between `yarn 3.1.1` and `jest` (see [jest/issues/12366](https://github.com/facebook/jest/issues/12366)), but until this is resolved we can install the `source-map-support` library directly to prevent the error.

## Changes

- Add `source-map-support` top-level dependency
- Change `yarn clean` so that success/failure of `rm -rf **/dist` doesn't affect whether second half runs



## Steps to Test

1. With current latest from `develop`, attempt to run a single test after running `yarn && yarn setup` (ex. `yarn test packages/sources/dxfeed/test/integration/adapter.test.ts`). This should result in the error above.
2. With this branch checked out, run `yarn && yarn setup` then repeat the same test. This time it should pass.

## Quality Assurance
